### PR TITLE
[17.0][MIG] purchase_order_line_menu

### PR DESCRIPTION
**Pull request by ThiagoMForgeFlow**
@ForgeFlow 

Rebase from https://github.com/OCA/purchase-workflow/pull/2253

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by gbrito — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ repos:
     hooks:
       - id: whool-init
   - repo: https://github.com/oca/maintainer-tools
-    rev: 9a170331575a265c092ee6b24b845ec508e8ef75
+    rev: 400ffa99242c8b225ab4d34de78721a68b292a61
```
Comment (HviorForgeFlow) [inline .pre-commit-config.yaml]: Not sure about this change.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**